render/container.go

```diff
@@ -35,23 +34,31 @@ var ContainerRenderer = MakeFilter(
 			ProcessRenderer,
 		),
 
-		// This mapper brings in short lived connections by joining with container IPs.
-		// We need to be careful to ensure we only include each edge once.  Edges brought in
-		// by the above renders will have a pid, so its enough to filter out any nodes with
-		// pids.
-		ShortLivedConnectionJoin(SelectContainer, MapContainer2IP),
+		// This mapper brings in connections by joining with container
+		// IPs.  We need to be careful to ensure we only include each
+		// edge once.  Edges brought in by the above renders will have
+		// a pid, so its enough to filter out any nodes with pids.
```

Comment:
Since we are re-writing the comment:
* It's not obvious to the reader why filtering out PIDs will ensure we get no duplicates (i.e. some explanation about PID-based connections vs conntrack may be in order)
* Most importantly, it's not obvious why duplicates are a problem (i.e. merge would settle it  ... except for counts)